### ISSUE TITLE
Ouais, c'est stylé de sauter des évaluations nazes et tout, mais sinon Voltaire, c'est grave stylé à faire quand il n'y a pas la pression scolaire pour combler les lacunes de l'école défaillante. Le truc, c'est que c'est payant. À quand un fork du Projet Voltaire lui-même ?

### ISSUE THREAD

**Issue by bphd** [Sshinx/Voltaire-is-Over#2]

**Comment by SikinoGaming** [comment]
C'est tout bonnement impossible, le code est gardé privé et un fork est donc impossible.

**Comment by bphd** [comment]
Ces projets souvent, pour tricher, tirent l'essence même du Projet: sa base de données. Si tu as déjà une liste de faute et leur corrections (voire les phrase complètes) on est jamais qu'à un HTML et un peu de CSS d'effectivement le forker. Je trouve que le plus dur est fait, à savoir trouver une méthode d'extraction de la substance savante du service

**Comment by SikinoGaming** [comment]
Je suis d'accord, sauf que le code est à la disposition de tout le monde que dans de rares circonstances, justement pour éviter ce genre d'action qui peuvent nuire au projet.

**Comment by bphd** [comment]
Oui, ils ferment la source pour éviter la concurrence et même la triche. En attendant tout ces projets ont effectivement extrait l'essence de Voltaire pour pouvoir tricher. Donc autant le forker quoi. La langue française c'est ouvert. Ils pourront pas rendre caduque le fork en refaisant une autre sauce maison fermée. Le fork sera toujours utile une fois fait car il apprends sur le français à la manière de Voltaire, donc c'est super pertienent et Voltaire pourra rien y faire de fou contre
